docs/morseSmaleQuadrangulation.md

```diff
@@ -0,0 +1,76 @@
+# Morse-Smale Quandrangulation example
+
+<!--[![Morse-Smale Quadrangulation example video tutorial](https://topology-tool-kit.github.io/img/gallery/morseSmaleQuadrangulation.jpg)](https://www.youtube.com/watch?v=eNW8l5PlgpU)-->
+
+<iframe width="100%" height="420"
+src="https://www.youtube.com/embed/eNW8l5PlgpU" frameborder="0"
+allowfullscreen></iframe>
+
+## Pipeline description
+
+This example first loads a mecanical model as a 2D triangle mesh from
+disk. This mechanical model embeds a vector field with the output of
```

Comment:
a vector field -› a collection of scalar fields
(usually, a vector field on a 2D surface is understood to have only 3 components, x y and z)